### DISCUSSION TITLE
File not found exception for hello camera and hello triangle sample

### DISCUSSION THREAD

**Discussion by BraveEvidence** [google/filament#4243] [Q&A]
I am trying out the hello camera and hello triangle sample but keep getting File not found exception for the materials folder. I have created materials folder but not sure why its not working. I also created assets folder via android studio and placed the materials in those folder and edited the code accordingly but still getting File not found exception.

**Comment by romainguy** [comment]
You don't need to create anything, the materials are built by the build system but you first need to build Filament itself. It should be described in the readmes.

**Reply by BraveEvidence** [reply]
@romainguy  Thanks for the answer. I am trying to create my own android project. Thats when the issue occurs. Also i noticed in the Hello camera sample the materials folder contains "lit.mat" but in the MainActivity it is loading "lit.filamat" from the materials folder. Is this correct?

**Reply by romainguy** [reply]
lit.mat is the source code of the material, it gets compiled to lit.filamat using the tool called "matc" that you can download from our releases page.

You can check this [standalone example](https://github.com/romainguy/sample-materials-shop)

**Reply by BraveEvidence** [reply]
@romainguy  Thanks
